webapp/src/store/boards.ts

```diff
@@ -25,23 +25,23 @@ type BoardsState = {
 export const fetchBoardMembers = createAsyncThunk(
     'boardMembers/fetch',
     async ({teamId, boardId}: {teamId: string, boardId: string}, thunkAPI: any) => {
+        const allTeamUsers = await octoClient.getTeamUsers()
+        console.log(allTeamUsers)
```

Comment:
Doesn't look like `allTeamUsers` is used, except for log message. These two lines can be removed.

Reply:
Done.